com.viber.Viber.json

```diff
@@ -0,0 +1,66 @@
+{
+  "app-id": "com.viber.Viber",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "viber",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--filesystem=home",
```

Comment:
Does it really need full access to home?

Reply:
It certainly needs access to ~/.Viber-PC to save it's config and it needs the ability to upload and save arbitrary files. I'd be happy to take suggestions for a smaller scope.

Reply:
Well you might be able to do something like override `$HOME` so it at least saves the config into its private directory. Keeping the access outside of that is probably fine though.